dashboard/test/ui/features/star_labs/applab/data_tab.feature

```diff
@@ -1,5 +1,6 @@
 @no_mobile
 @as_student
+@skip
```

Comment:
Is it possible to skip just the failing scenario?

Reply:
I'd also appreciate a comment explaining why we're disabling it and making it explicit when it can be reenabled, for discoverability